### PR TITLE
MOTOR-1068 Tornado no longer supports Python 3.7

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

**Comment by blink1073** [comment]
The Ubuntu 22.04 images do not have MongoDB installed, we'll need to use https://github.com/supercharge/mongodb-github-action#usage.

**Comment by blink1073** [comment]
The flake8 url in `.pre-commit-config.yaml` also needs to be updated to `https://github.com/pycqa/flake8`

**Comment by ShaneHarvey** [comment]
Let's open a new ticket for the github actions change.

File: .evergreen/config.yml [hunk 1]
```diff
@@ -807,10 +807,6 @@ axes:
         variables:
           TOX_ENV: "tornado6-pypy37"
           PYTHON_BINARY: "/opt/python/pypy3.7/bin/pypy3"
-      - id: "tornado6-py37"
-        variables:
-           TOX_ENV: "tornado6-py37"
-           PYTHON_BINARY: "/opt/python/3.7/bin/python3"
```
Comment (ShaneHarvey) [inline .evergreen/config.yml:813]: Why remove this?

File: .evergreen/config.yml [hunk 2]
```diff
@@ -892,10 +884,6 @@ axes:
         variables:
           TOX_ENV: "asyncio-py37"
           PYTHON_BINARY: "c:\\python\\Python37\\python.exe"
-      - id: "tornado6-py37"
```
Comment (ShaneHarvey) [inline .evergreen/config.yml:895]: Why remove this?

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

**Comment by blink1073** [comment]
I think we should merge a fix for https://jira.mongodb.org/browse/MOTOR-1073 before merging this one.

**Review by ShaneHarvey — APPROVED** [review]
No need to wait since MOTOR-1073 is unrelated.

**Pull request merged**